py/desiutil/depend.py

```diff
@@ -61,18 +61,14 @@
 #
 # default possible dependencies to check in add_dependencies()
 #
-possible_dependencies = [
-    'numpy', 'scipy', 'astropy', 'yaml', 'matplotlib',
-    'requests', 'fitsio', 'h5py', 'mpi4py', 'psycopg2', 'healpy',
-    'desiutil', 'desispec', 'desitarget', 'desimodel', 'desisim', 'desisurvey',
-    'specter', 'speclite', 'specsim', 'surveysim', 'redrock', 'desimeter',
-    'fiberassign', 'gpu_specter',
-    ]
-possible_envvars = [
-    'DESI_ROOT', 'DESI_SPECTRO_DATA', 'DESI_SPECTRO_REDUX', 'SPECPROD',
-    'DESI_SPECTRO_CALIB', 'DESI_BASIS_TEMPLATES',
-    'DESI_TARGET', 'DESIMODEL',
-    ]
+possible_dependencies = ['numpy', 'scipy', 'astropy', 'yaml', 'matplotlib',
+                         'requests', 'fitsio', 'h5py', 'mpi4py', 'psycopg2', 'healpy',
+                         'desiutil', 'desispec', 'desitarget', 'desimodel', 'desisim', 'desisurvey',
+                         'specter', 'speclite', 'specsim', 'surveysim', 'redrock', 'desimeter',
+                         'fiberassign', 'gpu_specter',]
+possible_envvars = ['DESI_ROOT', 'DESI_SPECTRO_DATA', 'DESI_SPECTRO_REDUX', 'SPECPROD',
+                    'DESI_SPECTRO_CALIB', 'DESI_BASIS_TEMPLATES',
+                    'DESI_TARGET', 'DESIMODEL',]
```

Comment:
For the record: Personally I don't prefer this formatting style change due to the extra indentation and editor columns needed.  I'm fine with you owning the style for desiutil and making this update here, but I don't want us to start enforcing this in other desi packages if the original author used the former style.

Reply:
That's fine. The other aspect is that desiutil already has so few style problems that it is easy to be strict. When I turned on the indentation enforcement there were less than 10 things to fix, so it is not a drain on productivity.